playbooks/infra/deploy-bm-hypervisor.yml

```diff
@@ -185,13 +185,15 @@
         type: bridge
         conn_name: bridge-baremetal
         method4: manual
-        method6: disabled
+        method6: "{{ 'manual' if (net_config | from_yaml)['ip6'] is defined else 'disabled' }}"
```

Comment:
Should we do the same for IPv4? just in case we'll need IPv6 only

Reply:
As far as I know we don't have such use case. This playbook just deploys hypervisor.